src/transport_ofi.c

```diff
@@ -271,6 +272,31 @@ typedef enum{
 /* default CQ depth */
 uint64_t shmem_transport_ofi_max_poll = (1ULL<<30);
 
+
+/* STX control variables */
+enum stx_share_alg_t {
+    ROUNDROBIN = 0,
+    RANDOM
+};
+typedef enum stx_share_alg_t stx_share_alg_t;
+stx_share_alg_t shmem_transport_stx_share_alg;
```

Comment:
static?

Reply:
Naming: we use ```shmem_transport_*``` for things that are part of the generic transport API (i.e. provided by both Portals and OFI).  For OFI-only things, we should name them ```shmem_transport_ofi_*```.